src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$card-width: 200px;
+$card-height: 300px;
+$cards-in-row: 4;
+$cards-gap: 48px;
```

Comment:
It's good practice, but it's just when you use this veriables more than one time

Reply:
So, this block can be reused in many places, and instead of searching all of them you can just replace a value in variable